docs/mctpd.md

```diff
@@ -3,29 +3,94 @@
 ## D-Bus
 
 `mctpd` provides a D-Bus service named `au.com.codeconstruct.MCTP1`, and a base
-object path of `/au/com/codeconstruct/mctp1`. For each known MCTP endpoint,
-`mctpd` will populate an object at
-`/au/com/codeconstruct/mctp1/networks/<NetworkId>/endpoints/<EID>`. The objects have
-interface `xyz.openbmc_project.MCTP.Endpoint`, as per [OpenBMC
-documentation](https://github.com/openbmc/phosphor-dbus-interfaces/tree/master/yaml/xyz/openbmc_project/MCTP).
+object path of `/au/com/codeconstruct/mctp1`. This tree represents both the
+local MCTP stack state and the results of remote device enumeration.
+
+```
+Service au.com.codeconstruct.MCTP1:
+└─ /au
+  └─ /au/com
+    └─ /au/com/codeconstruct
+      └─ /au/com/codeconstruct/mctp1
+        ├─ /au/com/codeconstruct/mctp1/interfaces
+        │ ├─ /au/com/codeconstruct/mctp1/interfaces/lo
+        │ └─ /au/com/codeconstruct/mctp1/interfaces/mctpi2c1
+        └─ /au/com/codeconstruct/mctp1/networks
+          └─ /au/com/codeconstruct/mctp1/networks/1
+            └─ /au/com/codeconstruct/mctp1/networks/1/endpoints
+              ├─ /au/com/codeconstruct/mctp1/networks/1/endpoints/8
+              └─ /au/com/codeconstruct/mctp1/networks/1/endpoints/10
+```
+
+
+## Top-level object: `/au/com/codeconstruct/MCTP1`
+
+This object serves as the global MCTP daemon namespace; it doesn't contain
+much at present, but hosts two trees of MCTP objects:
+
+ * Interfaces: Local hardware transport bindings that connect us to a MCTP bus
```

Comment:
"those connect" doesn't make sense (unless it's a separate clause) but "which connect" might be more correct.

However, "which" vs. "that" is never obvious to me, so referring to to Strunk & White:

> *That* is the defining, or restrictive, pronoun, *which* is the nondefining or nonrestrictive
> 
> > The lawnmower that is broken in the garage. (Tells which one)
>
> > The lawnmower, which is broken, in the garage (Adds a fact about the only mower in question)

So, we want _that_ here, because we're selecting a specific bus from a set in this case. TIL.